app/src/main/java/com/example/happymeals/ingredient/Ingredient.java

```diff
@@ -19,6 +19,7 @@ public class Ingredient extends DatabaseObject {
     private String unit;
     private String category;
     private Boolean needsUpdate = false;
+    private String id;
```

Comment:
Remove this, the parent class in in charge of it and some classes may rely on the parent. No reason to duplicate it. Parent will store and create it, also has the getId() already